inc/namespace.php

```diff
@@ -50,3 +55,59 @@ function load_publication_checklist() {
 
 	require_once Altis\ROOT_DIR . '/vendor/humanmade/publication-checklist/plugin.php';
 }
+
+/**
+ * Load Yoast Duplicate Posts, if enabled.
+ */
+function load_duplicate_posts() {
+	$config = Altis\get_config()['modules']['workflow']['clone-republish'] ?? null;
+
+	// Bail if Clone & Republish is disabled.
+	if ( ! $config ) {
+		return;
+	}
+
+	// Load the main plugin file.
+	require_once Altis\ROOT_DIR . '/vendor/yoast/duplicate-post/duplicate-post.php';
+}
+
+/**
+ * Remove the Duplicate Post settings page.
+ */
+function remove_duplicate_post_admin_page() {
+	remove_submenu_page( 'options-general.php', 'duplicatepost' );
```

Comment:
Changing/defining the post types was the option that it seemed most likely we'd want to hook into or customize. Of the other options, it depends hope deep down the rabbit hole we want to go. The default settings it saves on install/update are a good baseline of functionality (the update notice is hidden by default) and those settings could potentially be customized at the project level. I ultimately decided to use a lighter touch in a choice between things we'd most likely need and the kitchen sink. Happy to reconsider, though. Do you feel like the other options would be worth exposing?

Each option in the plugin has a corresponding field in the options table, so toggling any of these would be pretty simple, although it would mean doing a bunch of individual database checks (rather than a single `get_option` to get all the things), which was another reason I wasn't anxious to go too deep down the rabbit hole for that. Only the post types enabled option can be filtered (although I suppose we could propose other option filters and push them upstream if we wanted).

![Screen Shot 2021-05-25 at 1 44 28 PM](https://user-images.githubusercontent.com/991511/119559234-a2743680-bd5f-11eb-8883-702495ba870b.png)

Reply:
I think taxonomies and roles are probably the other 2 useful ones I can think of.

Remember that options themselves can be filtered eg. `add_filter( 'pre_option_<option>', ... )` so upstream changes aren't necessary, nor is updating database options directly.

I'd be tempted to restrict the capability to editor and above by default, (I don't think authors can publish right?).

Reply:
I had responded to this but it must've gotten eaten at some point. By default, authors can publish their own posts but not other people's posts. But I've restricted access to duplicate posts to those who _can_ publish other people's posts, so authors don't potentially run into a situation where they can clone a post belonging to someone else but can't edit it (since the author would be cloned as well).